src/ui/Windows/res/PasswordSafe3.rc2

```diff
@@ -963,6 +964,8 @@ BEGIN
   IDS_EDIT_NAMED_POLICY        "Change/View a Database Named Password Policy"
   IDS_TESTPOLICY               "Generate Password using selected Password Policy"
   IDS_CLICKTOCOPYGENPSWD       "Click to copy generated password to clipboard"
+  IDS_COPYPASSWORDHELP1        "Position 1 is the first character, 2 is the second etc. up to N for the last character of a password of length N. -1 is the last character, -2 the next to last etc. up to -N for the first password character. Positions must be separated by either spaces, commas or semi-colons to be recognised. There must also be a space, comma or semi-colon at the end of the list."
```

Comment:
I am only documenting the current process. But yes - how else does the program know that, in your example, the user wants the 5th character rather than the 50th?  OK - unlikely for this case but what if the user typed just "2;3;1" for a 12 character password? Is the '1' the first or maybe the 10th, 11th or 12th? (Some of my online accounts do not have the characters in ascending order and so "2 7 1" is possible as would be "6 2 10".